types/edtr-io__mathquill/index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for @edtr-io/mathquill 0.11
```

Comment:
`@edtr-io/mathquill` is too complicated for me. Unfortunately, I cannot fully verify it.

Reply:
What is the next step then?

Reply:
I tested the types with my application (which uses most of mathquill's properties) and it works perfectly. All it is, is a textarea that allows laTex input and allows user to input math equations. For example this application uses the mathquill field: https://www.desmos.com/calculator. And defined the types based on the documentation for mathquill: http://docs.mathquill.com/en/latest/Api_Methods/

Reply:
So what now haha